### PR TITLE
Replace Centos references by Rocky Linux

### PR DESCRIPTION
**Pull request by khaddict**

### CODE REVIEW

**Comment by nzlosh** [comment]
@khaddict The docker build has been updated to use the correct tags, can you push an empty to trigger the build process for this PR please?

File: packagingbuild/README.md [hunk 1]
```diff
@@ -6,8 +6,7 @@ Docker images used to build `.deb` and `.rpm` StackStorm packages in [StackStorm
 In these containers build environment specific for each OS distro is pre-installed and respective StackStorm packages are built for each platform.
 
 [`Dockerfiles` sources](https://github.com/StackStorm/st2packaging-dockerfiles/blob/master/packagingbuild):
-- CentOS 8
-- CentOS Stream 9
+- Rocky Linux 8
```
Comment (amanda11) [inline packagingbuild/README.md:9]: Should also add Rocky Llinux 9

**Review by amanda11 — CHANGES_REQUESTED** [review]

File: packagingbuild/README.md (hunk 1 above)
Comment (khaddict) [reply packagingbuild/README.md:10]: Rocky Linux 9 added

**Comment by nzlosh** [comment]
The builds are passing now that the Jammy fix was merged to master.  @amanda11 the image builds are automated from what I can tell, no need to push them manually https://hub.docker.com/repository/docker/stackstorm/packagingtest/general

**Comment by amanda11** [comment]
> The builds are passing now that the Jammy fix was merged to master. @amanda11 the image builds are automated from what I can tell, no need to push them manually https://hub.docker.com/repository/docker/stackstorm/packagingtest/general

OK - that's good. I had notes that previously we had lost the ability to do this due to dockerhub changes - but if its back again that's good.

**Review by amanda11 — APPROVED** [review]
LGTM - thanks for the contribution

**Pull request merged**